### ISSUE TITLE
Grav check_process still failing upgrade tests

### ISSUE THREAD

**Issue by tituspijean** [YunoHost-Apps/grav_ynh#32]
From https://ci-apps.yunohost.org/ci/job/5269 :

```
>> Upgrade from Version 1.3.1... [Test 5/11]

Preliminary install...
Installation failed. (1)
1641 Invalid choice for argument 'is_public', it must be one of yes, no, y, n, 1, 0
...
2019-10-28 18:32:21,136 INFO     moulinette.actionsmap process - processing action [920.1]: yunohost.app.install with args={'force': True, 'app': 'grav_ynh_check', 'args': 'domain=sous.domain.tld&path=/path&admin=package_checker&language=fr_FR&is_public=PUBLIC&', 'label': None, 'no_remove_on_failure': False}
...
2019-10-28 18:32:22,090 ERROR    yunohost cli - Invalid choice for argument 'is_public', it must be one of yes, no, y, n, 1, 0
```

```
>> Upgrade from Version 1.5.3... [Test 6/11]

Preliminary install...
Installation failed. (1)
1536 Invalid choice for argument 'is_public', it must be one of yes, no, y, n, 1, 0
...
2019-10-28 18:32:52,452 INFO     moulinette.actionsmap process - processing action [1157.1]: yunohost.app.install with args={'force': True, 'app': 'grav_ynh_check', 'args': 'domain=sous.domain.tld&path=/path&admin=package_checker&language=fr_FR&is_public=PUBLIC&', 'label': None, 'no_remove_on_failure': False}
...
2019-10-28 18:32:53,351 ERROR    yunohost cli - Invalid choice for argument 'is_public', it must be one of yes, no, y, n, 1, 0

Deleting...
Deleting failed. (1)
```

While in `check_process` for both 1.5.1 and 1.5.3: `is_public="Yes" (PUBLIC|public=1|private=0)`

Is this because `Yes` does not match with neither `1` nor `0`, that makes `is_public` default to `PUBLIC`?
Why is this happening now, and not back then?
Should we skip upgrade tests from those older versions?

**Comment by Jibec** [comment]
Isn't this a regression from the YunoHost package check or core? You may 
ping Aleks about this. I think it's something we can't fix.

**Comment by tituspijean** [comment]
@alexAubin, we need your insight. ;)

**Comment by alexAubin** [comment]
Well I'm not sure what the hell happens but you can see that the value being fed to `is_public` is literally `PUBLIC` which is neither yes or nor or y or ...

```
2019-10-28 18:32:52,452 INFO     moulinette.actionsmap process - processing action [1157.1]: yunohost.app.install 
with args={
'force': True, 
'app': 'grav_ynh_check', 
'args': 'domain=sous.domain.tld
           &path=/path
           &admin=package_checker
           &language=fr_FR
           &is_public=PUBLIC
           &', 
'label': None, 
'no_remove_on_failure': False}
```

So I believe that's an issue in package check or similar ?

**Comment by tituspijean** [comment]
Then, while waiting for a fix, I suggest to remove the two "upgrade from commit" lines.

**Comment by kay0u** [comment]
Sorry, for now I only have an limited access to the internet, so it's hard to debug...

**Comment by tituspijean** [comment]
Bump :) What should we do about this issue guys? @kay0u @Jibec ?

**Comment by kay0u** [comment]
It should be fixed with #34 (tested on my side)
